src/gsi/balmod.f90

```diff
@@ -482,6 +488,19 @@ subroutine prebal_reg(cwcoveqqcov)
        wgvk(:,:,:)=zero
        if(lnobalance) agvk_lm(:,:)=zero
     endif
+
+!   zero out balance for using radar radial wind observation
+    if (l_decouple_sf_vp) then
+       write(6,'(1x,A20,I4,A60)')'PREBAL_REG:(mype=',mype,  &
+            '  zero out balance correlation matrices for vp.'
+       bvk(:,:,:)   = zero                ! sf and vp
+    endif
+    if (l_decouple_sf_tps) then
+        write(6,'(1x,A20,I4,A60)')'PREBAL_REG:(mype=',mype,  &
```

Comment:
Add if(mype==0) before write to avoid writing too many information in large parallel job.

Reply:
I added if(mype==0) according to your comment

https://github.com/NOAA-EMC/GSI/pull/142/files#diff-1076dacbadc54101e22edb75ec2ed1b2b00a19f3c91fba0262afe2787eb50d77R492-R507